src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -138,13 +138,14 @@ public function onKernelResponse(ResponseEvent $event)
              * For supporting sessions in php runtime with runners like roadrunner or swoole the session
              * cookie need to be written on the response object and should not be written by PHP itself.
              */
+            $phpSessionCookieOptions = session_get_cookie_params();
```

Comment:
Can we do an `array_merge` in the constructor: `$this->sessionOptions = array_merge(session_get_cookie_params(), $sessionOptions);`